test/common/basic.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const assert = require('assert')
+
+// This is really just to see if the common testing framework
+// between NodeJS and Electron works.
+describe('Basic', function () {
+  it('should be always true', function () {
+    assert(true)
+  })
+})
```

Comment:
(But yes, this should not be commited, you're right). So good to go on this one, my bad.